src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,32 @@
+$cardWidth: 200px;
+$media: (488px, 768px, 1024px);
+
+.catalog {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+
+  grid: auto-flow / $cardWidth;
+  gap: 48px;
+  padding: 40px 0;
```

Comment:
Adding more space between header and cards wasn't in the requirements of this task. Padding here affected the design. It won't matter for this specific task maybe but it would matter for bigger projects.

Reply:
I know I just didn't like the cards touching the bottom of he page :)

Reply:
I just realized paddings are probably not even necessary, but I was trying to center the cards with paddings and variables at first and forgot to remove them.